lib/ios/RNNBackButtonOptions.m

```diff
@@ -11,11 +11,36 @@ - (void)applyOn:(UIViewController *)viewController {
 		
 		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithImage:image style:UIBarButtonItemStylePlain target:nil action:nil];
 		[self setBackItem:backItem onViewController:viewController];
-	} else if (self.title) {
-		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithTitle:self.title
+	} else {
+		NSString *title;
+		
+		title = self.title ? self.title : viewController.navigationItem.title;
+		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithTitle:title
 																	 style:UIBarButtonItemStylePlain
 																	target:nil
 																	action:nil];
+
+		if (self.color) {
+			[backItem setTintColor:[RCTConvert UIColor:self.color]];
+		}
+
+		if(self.fontFamily || self.fontSize) {
+			NSNumber* fontSize = self.fontSize;
```

Comment:
Also here: `NSNumber* fontSize = self.fontSize ? self.fontSize : [[NSNumber alloc] initWithInt: 18];`

Reply:
I prefer this for better readability frankly. This works.